app/uk/gov/hmrc/hec/services/scheduleService/HecTaxCheckScheduleService.scala

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.hec.services.scheduleService
+
+import cats.data.EitherT
+import cats.implicits._
+import com.google.inject.{ImplementedBy, Inject}
+import uk.gov.hmrc.hec.models
+import uk.gov.hmrc.hec.models.HECTaxCheck
+import uk.gov.hmrc.hec.services.{LockKeeperService, TaxCheckService}
+import uk.gov.hmrc.hec.util.Logging
+import uk.gov.hmrc.http.HeaderCarrier
+
+import javax.inject.Singleton
+import scala.concurrent.Future
+
+@ImplementedBy(classOf[HecTaxCheckScheduleServiceImpl])
+trait HecTaxCheckScheduleService {
```

Comment:
would something like `HECTaxCheckExtractionRunnerService` make more sense here? `HecTaxCheckScheduleService` sounds like this is going to be the thing which is going to be doing that scheduling rather than the actual work. Or maybe what is `HECTaxCheckExtractionService` now should be `HecTaxCheckScheduleService` and what is now `HecTaxCheckScheduleService` should be `HECTaxCheckExtractionService`

Reply:
done